lib/components/Toolbar.js

```diff
@@ -101,20 +103,42 @@ const Toolbar = ({
                 />
             ))}
         </ToolbarGroup>
+
+        <ToolbarGroup>
```

Comment:
@thibaudcolas Shouldn't the whole `ToolbarGroup` be wrapped into the `showUndoRedoControls` check? Does it displays fine if the group is empty?

Reply:
Yes, the `ToolbarGroup` contains some magic so it doesn't render anything when it has no children: https://github.com/springload/draftail/blob/60f2f6ef5684c10c7c409a6333f2b157b955fa45/lib/components/ToolbarGroup.js#L4-L7